scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -26,6 +26,33 @@ import org.ossreviewtoolkit.clients.fossid.model.summary.Summarizable
 import org.ossreviewtoolkit.model.CopyrightFinding
 import org.ossreviewtoolkit.model.LicenseFinding
 import org.ossreviewtoolkit.model.TextLocation
+import org.ossreviewtoolkit.utils.spdx.toSpdx
+
+/**
+ * Collection of problematic FossID license expressions.
```

Comment:
It would be helpful to have some information in the comment from where these licenses come (how was the list compiled) and whether it might have to be updated for new FossID versions (if this is the case).